core_v2/sources/v2_config.move

```diff
@@ -31,57 +30,79 @@ module aptos_names_v2::v2_config {
     /// The number of seconds after a name expires that it can be re-registered
     const CONFIG_KEY_REREGISTRATION_GRACE_SEC: vector<u8> = b"reregistration_grace_sec";
 
-    const DOMAIN_TYPE: vector<u8> = b"domain";
-    const SUBDOMAIN_TYPE: vector<u8> = b"subdomain";
-
     const DOMAIN_COLLECTION_NAME: vector<u8> = b"Aptos Domain Names V2";
     const SUBDOMAIN_COLLECTION_NAME: vector<u8> = b"Aptos Subdomain Names V2";
 
     /// Raised if the signer is not authorized to perform an action
     const ENOT_AUTHORIZED: u64 = 1;
     /// Raised if there is an invalid value for a configuration
     const EINVALID_VALUE: u64 = 2;
+    /// Domain length is invalid, it must be at least 3 characters
+    const EINVALID_DOMAIN_LENGTH: u64 = 3;
 
     const SECONDS_PER_YEAR: u64 = 60 * 60 * 24 * 365;
+    const SECONDS_PER_DAY: u64 = 60 * 60 * 24;
 
     struct Config has key, store {
```

Comment:
good to put this under some sort of resource group for all ANS

also this shouldn't have store